cpp/tests/interop/dlpack_test.cpp

```diff
@@ -225,8 +225,8 @@ TEST_F(DLPackUntypedTests, UnsupportedBroadcast1DTensorFromDlpack)
   constexpr int ndim = 1;
   // Broadcasted (stride-0) 1D tensor
   auto const data       = cudf::test::make_type_param_vector<T>({1});
-  int64_t shape[ndim]   = {5};
-  int64_t strides[ndim] = {0};
+  int64_t shape[ndim]   = {5};  // NOLINT
+  int64_t strides[ndim] = {0};  // NOLINT
```

Comment:
What does `NOLINT ` do?

Reply:
It disables clang-tidy checks on this line. I used it for structs containing raw C-style arrays when the structs were either 1) implementing what seemed like a well-defined protocol or file format (for I/O) or 2) when the structs looked like they could be memcopied over to device.